package.json

```diff
@@ -10,7 +10,7 @@
   },
   "license": "MIT",
   "engines": {
-    "vscode": ">=1.1.10",
+    "vscode": ">=1.9.0",
```

Comment:
do we need to bump any vscode dependencies too?

Reply:
I have absolutely no experience dealing with `package.json` files, I just know I can't use the API I need for an upcoming PR without bumping this version number...

Reply:
ok